### PR TITLE
SNO+: Add version number to SR structure

### PR DESCRIPTION
**Pull request by jcaravaca**
Author: Javier Caravaca <jcaravaca@berkeley.edu>

### CODE REVIEW

File: Source/Experiments/SNOP/SNOPModel.m [hunk 1]
```diff
@@ -1969,8 +1995,9 @@ -(BOOL) saveStandardRun:(NSString*)runTypeName withVersion:(NSString*)runVersion
 
     //Build run table
     NSMutableDictionary *detectorSettings = [NSMutableDictionary dictionaryWithCapacity:200];
-    
+
     [detectorSettings setObject:@"standard_run" forKey:@"type"];
+    [detectorSettings setObject:[[NSNumber alloc] initWithInt:1] forKey:@"version"]; //Increase when table structure changes
```
Comment (tlatorre-uchicago) [inline Source/Experiments/SNOP/SNOPModel.m]: Need to release this if you alloc it. Is it necessary to use an alloc here? Can you just create `[NSNumber numberWithInt:1]`?

File: Source/Experiments/SNOP/SNOPModel.m [hunk 2]
```diff
@@ -294,6 +294,9 @@ - (id) initWithCoder:(NSCoder*)decoder
     self.debugDBPort = [decoder decodeInt32ForKey:@"ORSNOPModelDebugDBPort"];
     self.debugDBIPAddress = [decoder decodeObjectForKey:@"ORSNOPModelDebugDBIPAddress"];
 
+    //Standard Runs
+    [self setStandardRunTableVersion:[[NSNumber alloc] initWithInt:STANDARD_RUN_VERSION]];
```
Comment (tlatorre-uchicago) [inline Source/Experiments/SNOP/SNOPModel.m]: same here.

**Comment by tlatorre-uchicago** [comment]
Did you get a chance to test this at the teststand?

File: Source/Experiments/SNOP/SNOPModel.m (hunk 2 above)
Comment (jcaravaca) [inline Source/Experiments/SNOP/SNOPModel.m]: This is released in the dealloc function

**Comment by jcaravaca** [comment]
Tested locally OK on my laptop but not yet at the TS. It should be fine since it doesn't use any hardware, but I'll test it tomorrow.

**Comment by tlatorre-uchicago** [comment]
merged.

**Pull request closed without merging**